cpp/tests/unary/legacy/unary_ops_test.cu

```diff
@@ -2383,16 +2383,18 @@ TYPED_TEST(gdf_logical_test, LogicalNot) {
 
 	std::vector<cudf::bool8> h_expect_v{colSize};
 
-	// compute NOT
-	for (cudf::size_type i = 0; i < colSize; ++i)
-		h_expect_v[i] = static_cast<cudf::bool8>( !h_input_v[i] );
+	std::transform(
+		std::begin(h_input_v),
+		std::end(h_input_v),
+		std::begin(h_expect_v),
+		[] (TypeParam e) -> cudf::bool8 {
+			return static_cast<cudf::bool8>(!e);
+		});
 
 	// Use vector to build expected output
-	auto expectCol = cudf::test::column_wrapper<cudf::bool8>{h_expect_v};
-
-	auto output = cudf::unary_operation(inputCol, cudf::unary_op::NOT);
-
+	auto output    = cudf::unary_operation(inputCol, cudf::unary_op::NOT);
 	auto outputCol = cudf::test::column_wrapper<cudf::bool8>(output);
+	auto expectCol = cudf::test::column_wrapper<cudf::bool8>{h_expect_v};
```

Comment:
`cudf::bool8` is the old one. Use `cudf::experimental::bool8`.

Reply:
@jrhemstad but this test is legacy.

Reply:
Yea, sorry. This was my OCD not liking the raw for loop :)